readme_template.md

```diff
@@ -50,26 +50,31 @@ Host file source | Description | Home page | Raw hosts | Update frequency | Lice
 
 
 ## Extensions
-The unified hosts file is optionally extensible.  Extensions are used to include domains by category.  Currently we offer the following categories: `fakenews`, `social`, `gambling`, and `porn`.
 
-Extensions are optional, and can be combined in various ways wth the base hosts file.  The combined products are stored in the [`alternates`](https://github.com/StevenBlack/hosts/tree/master/alternates) folder.
+The unified hosts file is optionally extensible.  Extensions are used to include domains by category.  Currently, we offer the following categories: `fakenews`, `social`, `gambling`, and `porn`.
```

Comment:
You force Steve @StevenBlack to update that line at each inclusion of a new alternate.

Reply:
Not sure I get what you mean. The lines are the same. I just made sure there's newlines around headings.

Reply:
>  Currently, we offer the following categories: `fakenews`, `social`, `gambling`, and `porn`.

It can be easily forgotten if a new category is included.

Reply:
But I didn't change this text, this is how it is in master too.

Reply:
Ohh my bad forget about this. Sorry, It was because of the side by side comparison :joy: